fortifyapi/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '3.1.16'
+__version__ = '3.1.17'
```

Comment:
```suggestion
__version__ = '3.1.18'
```

Reply:
Suggestion bc of version conflict, we may want to bump version to this after we merge PR 64